frontend/src/components/common/SideBar/index.tsx

```diff
@@ -67,6 +67,7 @@ export default function SideBar({
     }
   };
 
+  // TODO: Need help fixing this
```

Comment:
@seve I might need your help to fix this hook, since I'm a little unclear how `forceOpen` is intended to be used 😆 Thank you!

Reply:
forceOpen is just used to check for uniqueness, upstream it's changed to denote a change in target cellType for the cell info sidebar